distribution/docs/src/main/resources/content/_integrating/_endpoints/catalog-rest-endpoint.adoc

```diff
@@ -3,291 +3,329 @@
 :status: published
 :operations: ingest, retrieval, crud
 :link: _catalog_rest_endpoint
-:url: \${public_url}/services/catalog
-:summary: Allows clients to perform operations on the Catalog using REST, a simple architectural style that performs communication using HTTP.
-:implements: https://www.w3.org/2001/sw/wiki/REST[REST specification] {external-link}.
+:summary: Uses REST to interact with the catalog.
 
 == {title}
 
 The Catalog REST Endpoint allows clients to perform operations on the Catalog using REST, a simple architectural style that performs communication using HTTP. 
 
-.Catalog REST Endpoint Operations
-The Catalog REST Endpoint provides the capability to query, create, update, and delete metacards and associated resources in the catalog provider.
-
-Any web browser can be used to perform a REST read.
-Various other tools and libraries can be used to perform the other HTTP operations on the REST endpoint (e.g., soapUI, cURL, etc.)
-
 Bulk operations are not supported: for all RESTful CRUD commands, only one metacard ID is supported in the URL.
 
-Operations on the REST endpoint can be performed as follows:
-
-.Catalog REST Endpoint Operations
-[cols="2,1m,2,5m", options="header"]
-|===
+The Catalog REST Endpoint can be used for one or more of these operations on an instance of ${branding}:
 
-|Operation
-|HTTP Request Type
-|Details
-|Example URL
+* <<{integrating-prefix}catalog_rest_create_operation_examples,Ingest metacards or resources into the ${branding} catalog.>>
+* <<{integrating-prefix}catalog_rest_read_operation_examples,Retrieve metacards or resources from the catalog.>>
+* <<{integrating-prefix}catalog_rest_update_operation_examples,Update metacards or resources in the catalog.>>
+* <<{integrating-prefix}catalog_rest_delete_operation_examples,Delete resources and metadata from the catalog.>>
+* <<{integrating-prefix}catalog_rest_sources_operation_examples,Get information about configured sources.>>
 
-|<<{integrating-prefix}catalog_rest_create_example,create>>
-|POST
-|HTTP request body contains the input to be ingested.
+This example metacard can be used to test the integration with ${branding}.
 
-`<input transformer>` is the name of the transformer to use when parsing metadata (optional).
+.Example Metacard
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<metacard xmlns="urn:catalog:metacard" xmlns:gml="http://www.opengis.net/gml" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:smil="http://www.w3.org/2001/SMIL20/" xmlns:smillang="http://www.w3.org/2001/SMIL20/Language" gml:id="3a59483ba44e403a9f0044580343007e">
+  <type>${branding-lowercase}.metacard</type>
```

Comment:
This may always be `ddf.metacard` and not substituted with branding. They may have a reason to have their `branding`.metacard but ddf.metacard should always be valid.

Reply:
fixed